### DISCUSSION TITLE
Climate TRACE

### DISCUSSION THREAD

**Discussion by paula-fel** [transport-data/community#9] [Sources]
Includes the following indicators:

- GHG emissions [tonnes]

Geography:

- worldwide

Relevant URLs:

- https://climatetrace.org/downloads

**Comment by dominic-sheldon** [comment]
It turns out that Johns Hopkins University are responsible for their transport data:
https://www.jhuapl.edu/PressRelease/221109-apl-creates-first-ever-automated-approach-to-estimate-road-transportation-greenhouse-gas-emissions
